src/internal/modal.ts

```diff
@@ -71,14 +71,26 @@ export default class Modal {
     if (event.key !== 'Tab' || this.isExternalActivated) return;
     if (!this.isActive()) return;
 
+    const elementsWithTabbableControls = [
+      "audio",
+      "video",
+      "iframe"
+    ]
+
+    const possiblyHasTabbableChildren = (element: HTMLElement) => {
+      return (
+        elementsWithTabbableControls.includes(element.tagName.toLowerCase())
+        || element.hasAttribute("controls")
+        // Should we add a data-attribute for people to set just in case they have an element where we don't know if it has possibly tabbable elements?
```

Comment:
This focus trapping logic is a constant game of whack-a-mole. We may find alternate solutions to iframes and such or we may [eventually] have a platform API for doing it. A data- attribute seems fairly benign, though. Would it be a space-delimited list of tag names? It might be useful for some custom elements. 🤔